frontend/src/slicer/detailsSlice.ts

```diff
@@ -0,0 +1,34 @@
+import {createSlice, PayloadAction} from '@reduxjs/toolkit';
+import {RootState} from '../app/store';
+import {IDetailsData} from "../interfaces/IThumbnailData";
+import {IDetailsState, intitialDetailsData} from "../interfaces/IStates";
+
+
+const initialState: IDetailsState = {
```

Comment:
cryptic name, inconsistent with initialDetailsData... might drop the predicate 'State' from all your State variable names.. then use InitialDetails... not sure... is it good to start with upper case? I thought that only components are capitalized.